data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -4239,6 +4239,11 @@ spec:
                       PreloadedOSImageName uses the named preloaded RHCOS image from PC/PE,
                       instead of create and upload a new image for each cluster.
                     type: string
+                  prismAPICallTimeout:
+                    description: |-
+                      PrismAPICallTimeout sets the timeout (in minutes) for the prism-api calls.
+                      If not configured, the default value of 5 minutes will be used as the prism-api call timeout.
```

Comment:
What is the default value? Here it is 5 and here https://github.com/openshift/installer/pull/9377/files#diff-8974fce42302d00ebfd38b7ffe6ef5d62a97ed2628f2004ae525fe2e4728aa01R31 it is 10.

Reply:
fixed